exp/api/v1beta1/ocimanagedmachinepool_types.go

```diff
@@ -21,6 +21,9 @@ type OCIManagedMachinePoolSpec struct {
 	// +optional
 	ProviderID *string `json:"providerID,omitempty"`
 
+	// Version represents the version of the OKE node pool.
```

Comment:
```suggestion
	// Version represents the kubernetes version of the OKE node pool.
```

I'm not 100% sure my change is correct, but I wanted to make sure it was as clear that the version is for kubernetes or something else. The way it read to me was some OKE node pool version and I'm not sure what that is.